libs/block-content/website/src/lib/teaser/teaser.tsx

```diff
@@ -196,32 +289,52 @@ export const Teaser = ({teaser, alignment, className}: BuilderTeaserProps) => {
 
   const {
     date,
-    elements: {H5, H6, Link, Image}
+    elements: {Link, Image}
   } = useWebsiteBuilder()
 
-  return (
-    <TeaserWrapper {...alignment}>
-      <Link color="inherit" underline="none" href={href} css={teaserLinkStyles}>
-        <TeaserInnerWrapper className={className}>
-          {image && <Image image={image} css={teaserImageStyles} />}
+  const linkStyles = useMemo(() => teaserLinkStyles(theme), [theme])
+  const enhancedImageStyles = useMemo(
+    () => getEnhancedImageStyles(theme, isHovered),
+    [theme, isHovered]
+  )
 
-          <TeaserContent>
+  return (
+    <TeaserWrapper
+      {...alignment}
+      onMouseEnter={() => setIsHovered(true)}
+      onMouseLeave={() => setIsHovered(false)}>
+      <Link color="inherit" href={href ?? ''} className={className} css={linkStyles}>
+        {image ? (
+          <ImageWrapperStyled>
+            <Image image={image} css={enhancedImageStyles} />
+          </ImageWrapperStyled>
+        ) : (
+          <ImagePlaceholder />
+        )}
+
+        <TeaserContentStyled>
```

Comment:
As I mentioned in the meeting already before, this now causes that you can not reorder the elements at will. Do not have a wrapper around the content. 

If you don't have a wrapper around the content you can use grid-template-area & grid-area to define how things are displayed.